routes/users.ts

```diff
@@ -11,5 +11,6 @@ const router = Router();
  * */
 // eslint-disable-next-line
```

Comment:
@prakashchoudhary07 , for now, disable that specific rule for this file. We can check that error later.